tests/ui/exe-trace/bool/main.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// kani-flags: --harness harness --enable-unstable --gen-exe-trace
+
+#[kani::proof]
+pub fn harness() {
+    let bool_1: bool = kani::any();
+    assert!(bool_1 != true);
+    // Note: We can't test a false value yet because any::<bool>() could be any non-zero number.
```

Comment:
Oh, if you want `any::<bool>()` to be pretty in the trace, you can change it to the following:

```rust
impl Arbitrary for bool {
  fn any() -> Self {
    let byte : u8 = crate::any();
    crate::assume(byte < 2);
    byte != 0
  }
}
```

Reply:
This change would be nice, but I think we have to run some performance tests before making it. I created #1490 to track this.